pkg/globalnet/controllers/ipam/constants.go

```diff
@@ -21,7 +21,7 @@ type Operation string
 
 const (
 	handlerResync          = time.Hour * 24
-	submarinerIpamGlobalIp = "submariner.io/globalIp"
+	submarinerIpamGlobalIP = "submariner.io/globalIP"
```

Comment:
We can't change the label name w/o changing it elsewhere and causing backwards compatibility and migration issues.  We should leave "_globalIp_" as is.

Reply:
Correct, the submariner.io/globalIp part needs to stay as it 's (the constant name can change, but not the content).